js/main.js

```diff
@@ -1,55 +1,106 @@
+*/
 
-returnRandomNumber(0, 30);
-//console.log(returnRandomNumber([2], [5]));
+const NAMES = [
+  'Константин',
+  'Олег',
+  'Михаил',
+  'Александра',
+  'Соня',
+  'Егор',
+  'Дмитрий',
+  'Яна',
+  'Ксения',
+  'Юлия',
+  'Злата',
+  'Сергей',
+  'Виталий',
+  'Ольга',
+  'Роман',
+  'Олег',
+  'Глеб',
+  'Марина',
+  'Ирина',
+  'Ева',
+  'Алиса',
+  'Анелина',
+  'Алина',
+  'Арина',
+  'Инга',
+];
 
-/* Функция, возвращающая случайное целое число из переданного диапазона включительно.
-диапазон может быть только положительный, включая ноль.
-А также придумайте, как функция должна вести себя, если передать значение «до» меньшее, чем значение «от», или равное ему.
-*/
+const SURNAMES = [
+  'Смирнов',
+  'Новиков',
+  'Петров',
+  'Максимова',
+  'Гаврилова',
+  'Ершов',
+  'Исаев',
+  'Осипова',
+  'Третьякова',
+  'Евдокимова',
+  'Аксёнов',
+  'Николаев',
+  'Моисеев',
+  'Лапина',
+  'Дьячков',
+  'Фролов',
+  'Одинцов',
+  'Трофимова',
+  'Нестерова',
+  'Филатова',
+  'Авдеева',
+  'Антонова',
+  'Назарова',
+  'Белова',
+  'Селезнёва',
+];
 
-function getRandomIntInclusive(min, max) {
-  if (min < 0 || max < 0) {
-    return 'Error';
-  }
+const MESSAGE = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!',
+];
 
-  min = Math.ceil(min);
-  max = Math.floor(max);
-  return Math.floor(Math.random() * (max - min + 1)) + min; //Максимум и минимум включаются
-}
-getRandomIntInclusive(3, 8);
-//console.log(getRandomIntInclusive(2, 8));
+const GENERATED_OBJECT = 25;
 
-/*
-  Функция для проверки максимальной длины строки.
-  Будет использоваться для проверки длины введённого комментария, но должна быть универсальна.
-*/
+const getRandomPositiveInteger = (min, max) => {
+  const lower = Math.ceil(Math.min(Math.abs(min), Math.abs(max)));
+  const upper = Math.floor(Math.max(Math.abs(min), Math.abs(max)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
 
-const maxSrtLength = (str, maxLength) => {
-  if (typeof str !== 'string') {
-    throw new TypeError('Не строка');
-  }
+//const checkStringLength = (string, length) => string.length <= length;
 
-  return str.length <= maxLength;
+const getRandomArrayElement = (elements) => elements[getRandomPositiveInteger(0, elements.length - 1)];
+
+const createRandomObject = () => ({
+  name: `${getRandomArrayElement(NAMES)  } ${  getRandomArrayElement(SURNAMES)  } ${  getRandomArrayElement(MESSAGE)}`,
```

Comment:
В целом у тебя всё в правильном направление, насколько я вижу, просто сейчас объект с одним полем `name` возвращается, а надо добавить все остальные поля тоже

```suggestion
  name: `${getRandomArrayElement(NAMES)  } ${  getRandomArrayElement(SURNAMES)  } ${  getRandomArrayElement(MESSAGE)}`,
  id: 1,
  url: 'photos/{{photo-1}}.jpg',
  description: 'photo1',
  likes: 2,
  comments: [
    {
      id: 135,
      avatar: 'img/avatar-6.svg',
      message: 'В целом всё неплохо. Но не всё.',
      name: 'Артём',
    }
  ]
```